tutorials/tour/basics.md

```diff
@@ -84,36 +115,25 @@ val getTheAnswer = () => 42
 println(getTheAnswer()) // 42
 ```
 
-If your function body spans across multiple lines, you must wrap them with `{}`.
-
-```
-val greet = (name: String) => {
-  println("Hello, " + name + "!")
-  println("How are you doing today?")
-}
-greet("Scala developer")
-// Hello, Scala developer!
-// How are you doing today?
-```
-
 We will cover functions in depth [later](anonymous-function-syntax.md).
 
 ## Methods
 
 Methods look and behave very similar to functions, but there are a few key differences between them.
 
-Methods are defined like below with the `def` keyword followed by its name, parameter(s), return type, and body.
+Methods are defined like below with the `def` keyword followed by its name, a list of parameter groups, return type, and an expression.
```

Comment:
It might be worth mentioning that (unlike functions) methods can also have *type* parameters.